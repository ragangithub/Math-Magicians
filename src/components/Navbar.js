import React from 'react';
import { Link } from 'react-router-dom';
import styles from '../styles/Navbar.module.css';

const Navbar = () => (
  <div className={styles.navbar}>
    <h1>Math Magicians</h1>
    <ul className={styles.nav}>
      <li>
        <Link to="/">Home</Link>
      </li>
      <div className={styles.border} />
      <li>
        <Link to="/calculator">Calculator</Link>
      </li>
      <div className={styles.border} />
      <li>
        <Link to="/quote">Quotes</Link>
      </li>
    </ul>
  </div>
);

export default Navbar;
