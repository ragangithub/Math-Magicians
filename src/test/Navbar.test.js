import React from 'react';
import { render } from '@testing-library/react';
import { MemoryRouter } from 'react-router-dom';
import Navbar from '../components/Navbar';

describe('Navbar component', () => {
  test('renders correctly', () => {
    const { container } = render(
      <MemoryRouter>
        <Navbar />
      </MemoryRouter>,
    );
    expect(container).toMatchSnapshot();
  });
});
