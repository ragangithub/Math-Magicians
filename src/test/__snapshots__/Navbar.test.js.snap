// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Navbar component renders correctly 1`] = `
<div>
  <div
    class="navbar"
  >
    <h1>
      Math Magicians
    </h1>
    <ul
      class="nav"
    >
      <li>
        <a
          href="/"
        >
          Home
        </a>
      </li>
      <div
        class="border"
      />
      <li>
        <a
          href="/calculator"
        >
          Calculator
        </a>
      </li>
      <div
        class="border"
      />
      <li>
        <a
          href="/quote"
        >
          Quotes
        </a>
      </li>
    </ul>
  </div>
</div>
`;
